python/pyspark/sql/conf.py

```diff
@@ -63,6 +63,12 @@ def _checkType(self, obj, identifier):
             raise TypeError("expected %s '%s' to be a string (was '%s')" %
                             (identifier, obj, type(obj).__name__))
 
+    @ignore_unicode_prefix
+    @since(2.4)
+    def isModifiable(self, key):
+        """Is the configuration property modifiable or not."""
```

Comment:
Why not make all doc  comments across languages identical?

Reply:
From which language would you prefer the comment? or maybe the comment should contain more information? Please, let me know if something is not clear, and we should explain that in the comment.

Reply:
I am going to change to:
```
Indicates that the configuration property with the given key modifiable in the current session or not.
```

Reply:
Minor language tweak suggestion:

```
Indicates whether the configuration property with the given key is modifiable in the current session.
```